array-methods.js

```diff
@@ -49,10 +62,25 @@ var datasetWithRoundedDollar = null;
     }
   assign the resulting new array to `roundedDime`
 */
-var datasetWithRoundedDime = null;
+var datasetWithRoundedDime = bankBalances.map(currentObject => {
+
+  let amount = currentObject.amount;
+  let state = currentObject.state;
+  let roundedDime = roundAndAdd(amount, undefined, 1);
+
+  return {
+    'amount': amount,
+    'state': state,
+    'roundedDime': roundedDime
+  }
+});
 
 // set sumOfBankBalances to be the sum of all value held at `amount` for each bank object
-var sumOfBankBalances = null;
+
+var sumOfBankBalances = parseFloat(bankBalances
+  .reduce((sum, currentObject) => {
+    return sum += parseFloat(currentObject.amount);
+  }, 0).toFixed(2));
```

Comment:
move the final closing `)` to the next line to differentiate the parseFloat from the reduce
```js
  }, 0).toFixed(2)
);
```